<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTestWindow</class>
 <widget class="QMainWindow" name="VideoTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="PhGraphicView" name="videoView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent">
     <property name="title">
      <string>Open recent</string>
     </property>
     <addaction name="actionEmpty"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionReverse"/>
    <addaction name="actionNext_frame"/>
    <addaction name="actionPrevious_frame"/>
    <addaction name="separator"/>
    <addaction name="actionSet_timestamp"/>
    <addaction name="actionGo_to"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisplay_media_panel"/>
    <addaction name="actionDeinterlace_video"/>
    <addaction name="actionUse_native_video_size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="text">
    <string>Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionNext_frame">
   <property name="text">
    <string>Next frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious_frame">
   <property name="text">
    <string>Previous frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionSet_timestamp">
   <property name="text">
    <string>Set timestamp...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="text">
    <string>Reverse</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </action>
  <action name="actionGo_to">
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionDisplay_media_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display media panel</string>
   </property>
  </action>
  <action name="actionDeinterlace_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deinterlace video</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUse_native_video_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use native video size</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhGraphicView</class>
   <extends>QWidget</extends>
   <header>PhGraphic/PhGraphicView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
